api/core/model_runtime/model_providers/spark/llm/_client.py

```diff
@@ -148,7 +148,9 @@ def gen_params(self, messages: list, user_id: str,
         data = {
             "header": {
                 "app_id": self.app_id,
-                "uid": user_id
+                # "uid": user_id
```

Comment:
Please remove this line. The comments is enough.

Reply:
@crazywoola Thanks for the review, removed it